unicoremc/models.py

```diff
@@ -173,6 +173,8 @@ class Project(models.Model):
         default=settings.MESOS_DEFAULT_MEMORY_ALLOCATION)
     marathon_instances = models.IntegerField(
         default=settings.MESOS_DEFAULT_INSTANCES)
+    marathon_health_check_path = models.CharField(
+        max_length=255, blank=True, null=True)
```

Comment:
We can once we know all apps have `/health/` stuff, at the moment they don't and I felt this could be a better way for us to test it without marathon taking stuff offline & rebooting it all the time because `/health/` was missing